packages/widget_toolkit_pin/test/lib_pin_code/view/pin_golden_test.dart

```diff
@@ -27,12 +29,31 @@ Widget pinCodePageFactory(
       backgroundColor: Colors.blue,
       body: Padding(
         padding: const EdgeInsets.all(30),
-        child: PinCodeKeyboard.generic(
-          isLoading: isLoading,
-          onAutoSubmit: (p0) {},
+        child: PinCodeKeyboard(
+          // isLoading: isLoading,
+          // onAutoSubmit: (p0) {},
           onChangePin: () {},
-          keyLength: 6,
+          keyLength: 6, pinCodeService:pinCodeService, biometricsLocalDataSource: ProfileLocalDataSource(),
           // pinCodeService: pinCodeService,
         ),
       ),
     );
+/// you can use this to store the value, for example in [SharedPreferences]
+class ProfileLocalDataSource implements BiometricsLocalDataSource {
```

Comment:
Do you see any issues in using shared preferences in golden tests ?

Reply:
Yes, I have deleted it